apache/superset
# feat: Moves Profile to Single Page App (SPA)

### SUMMARY
- Moves the Profile page to SPA
- Introduces a new endpoint `/profile` 
- Deprecates and adds a redirect to the old endpoint `/superset/profile` to preserve backward compatibility
- Organizes the files using the `pages` and `features` folders
- Removes old entrypoint related files

### TESTING INSTRUCTIONS
1 - Make sure the Profile page works as before but in the context of SPA (does not reload the page when navigating)
2 - Make sure that `/superset/profile` redirects to `/profile`

### ADDITIONAL INFORMATION
